### PR TITLE
Adds frontend support for multiple per exchange accounts

### LINKED ISSUE #47: Ability to add more than one account of an exchange
Even thought this might be an issue that won't affect many people, I think it is important to include the possibility for it early on in the development process so that data models etc allow for it.

### LINKED ISSUE #2850: Make exchanges name consistent
Created docker image from commit https://github.com/rotki/rotki/commit/a7ef8be31f8059eeff34509667688a87a63edfc6

http://localhost:8084/ page
Screen Shot 2021-05-05 at 7 35 24 PM

  http://localhost:8084/accounts-balances/exchange-balances/
Screen Shot 2021-05-05 at 7 35 54 PM

http://localhost:8084/history/deposits-withdrawals
Screen Shot 2021-05-05 at 7 36 39 PM

### PR DESCRIPTION
**Pull request by kelsos**
Closes #47 
Closes #2850

### CODE REVIEW

**Comment by kelsos** [comment]
Closes #47

**Review by LefterisJP — COMMENTED** [review]
Most of the things seem to work. Things that need check/fixes are:

### Changelog entry

Changelog entry for #47

### Error popup is missing data

Providing an invalid api key for bittrex exchange gave me this popup.



I would have expected to see the name "bittrex" after of.

### Kraken account type does not seem to be mirrored in the edit dialog

The kraken account type is always starter in the edit dialog.



## Exchange balances double counted in the dashboard

I have 2 different kraken accounts so I am a good candidate for testing this. After adding both kraken api keys I see double the amount of ETH (and any token) for **each** exchange in the dashboard. So let's say kraken1 -> `K`, kraken2 -> `L`. I see `2 * L + 2 * K` ETH.

What's interesting is that in the exchange balances, when I go to kraken it's correct. I see `L + K`.

What's even more interesting is that the wrong dashboard balances persist even after restart. So the mistake is not just during addition.

**Review by LefterisJP — APPROVED** [review]
Hey @kelsos I am pre-approving but two things still remain.

1. The changelog entry as I mentioned in the previous review.
2. Can you make the editability of the passphrase be the same as api key/secret. When I try to edit an exchange with a passphrase I would like to see the same green thing, meaning there is a passphrase input. Passphrases are not optional in any exchange (yet).

File: docs/changelog.rst [hunk 1]
```diff
@@ -2,10 +2,12 @@
 Changelog
 =========
 
+* :feature:`47` Users can now add multiple accounts per supported exchange.
 * :feature:`1881` Users can now access an about screen with information about the application.
 * :feature:`1549` Rotki premium users will now be able to switch to a dark mode and change the theme colors.
 * :feature:`1674` Add experimental support for BlockFi imports using CSV files.
 * :feature:`2224` Add experimental support for Nexo imports using CSV files.
+* :bug:`2850` User will now see a consistent naming of exchanges across the application.
```
Comment (LefterisJP) [inline docs/changelog.rst:10]: This din't really need a changelog but anyway doesn't hurt to have.

**Pull request merged**